spec/client_spec.js

```diff
@@ -597,10 +637,10 @@ describe('Client', () => {
         }).to.throw(Error)
       })
 
-      it('rejects the promise after 10 seconds', (done) => {
+      it('rejects the promise after the timeout length', (done) => {
```

Comment:
Should we also include a test to check that it doesn't reject the promise before the timeout length?

Reply:
I think the tests above should cover this as they test the promise is resolved and the appropriate data returned